http/context.go

```diff
@@ -195,66 +195,59 @@ func (context *Context) readForm(body []byte, target interface{}) error {
 	return context.valuesToObject(values, target)
 }
 
-func (context *Context) inspectModel(target interface{}) map[string]interface{} {
+func (context *Context) inspectModel(target interface{}) map[string]reflect.Type {
 	v := reflect.Indirect(reflect.ValueOf(target))
-	fieldMap := make(map[string]interface{}, v.NumField())
+	fieldMap := make(map[string]reflect.Type, v.NumField())
 	for i := 0; i < v.NumField(); i++ {
 		fieldMap[v.Type().Field(i).Name] = v.Field(i).Type()
 	}
 	return fieldMap
 }
 
+func (context *Context) valuesToArray(fieldType reflect.Type, values []string) []interface{} {
+	elemKind := fieldType.Elem().Kind()
+	anon := make([]interface{}, len(values))
+	coerce := func(v string) interface{} { return v }
+	if elemKind == reflect.Int {
+		coerce = func(v string) interface{} {
+			i, err := strconv.Atoi(v)
+			log.Warnf("error encountered coercing url value '%s' to int: %s", v, err)
+			return i
+		}
+	} else if elemKind == reflect.String {
+
```

Comment:
should there be a comment or code here?

Reply:
hmmm not sure what I was doing here, I will clean it up